jest.config.js

```diff
@@ -35,5 +35,6 @@ module.exports = {
     `./tests/unit/helpers/window_mock.js`,
     `./tests/unit/helpers/console_error_throw.js`,
     `jest-localstorage-mock`
-  ]
+  ],
+  cache: false
```

Comment:
is this necessary?

Reply:
for me, `yarn test:unit` fails without it

Reply:
So I think you should figure out why and not necessary disable caching for everybody. This will slow down local testing.